tests/test_workflow_api.py

```diff
@@ -581,12 +581,19 @@ async def test_import_export_workflow(
         assert "id" not in wftask["task"]
     assert res.status_code == 200
 
-    # Check that the exported workflow matches with the one in the original
-    # JSON file
+    # Check that the exported workflow is an extension of the one in the
+    # original JSON file
     wf_old = WorkflowExport(**workflow_from_file).dict(exclude_none=True)
     wf_new = WorkflowExport(**workflow_exported).dict(exclude_none=True)
-
-    assert wf_old == wf_new
+    debug(wf_old)
+    debug(wf_new)
+    for task_old, task_new in zip(wf_old["task_list"], wf_new["task_list"]):
```

Comment:
This does not catch task lists of different lengths. Pleas add an assertion that their lengths are the same, before this loop.